liquibase-core/src/main/java/liquibase/command/core/DropAllCommand.java

```diff
@@ -104,8 +104,7 @@ public CommandResult run() throws Exception {
         commandScope.addArgumentValue(InternalDropAllCommandStep.DATABASE_ARG, this.database);
```

Comment:
@filipelautert this class has been marked as deprecated, but this `run()` hasn't been. Should not we mark it as deprecated as well?